### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by himuura** [wazuh/wazuh-dashboard#357]
**Describe the bug**

Endpoint Summary -> (select the endpoint) -> On the MITRE ATT&CK mini-dashboard select one -> Resolves to "Something went wrong". (Internal Server Error) 

**To Reproduce**

Endpoint Summary -> (select the endpoint) -> On the MITRE ATT&CK mini-dashboard select one -> Resolves to "Something went wrong". (Internal Server Error)
Happened after migrating from 4.8.0 to 4.9.0

**Expected behavior**
Should open the MITTRE ATT&CK dashboard and that doesn't happen. It i use the separator MITRE ATT&CK on the top, it shows fine.

**OpenSearch Version**
Opensearch that ships with wazuh 4.9.0-2

**Dashboards Version**
Default dash that ships with 4.9.0-2.

**Plugins**

Nothing

**Screenshots**

![image](https://github.com/user-attachments/assets/121c1f6d-c271-4e84-b5ce-87d5d00067a6)
![image](https://github.com/user-attachments/assets/8ac60b1e-984f-453b-b705-ac32ebd804f6)

**Host/Environment (please complete the following information):**
 - OS:
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.5 LTS (Jammy Jellyfish)" ---> Wazuh Server
 - Browser and version [e.g. 22]
Any browser.

**Additional context**

Oct 11 16:50:18 srv-wazuh opensearch-dashboards[702]: {"type":"error","@timestamp":"2024-10-11T15:50:18Z","tags":[],"pid":702,"level":"error","error":{"message":"Internal Server Error",
"name":"Error","stack":"Error: Internal Server Error\n at HapiResponseAdapter.toError (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:127:19)\n at Hapi
ResponseAdapter.toHapiResponse (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:83:19)\n at HapiResponseAdapter.handle (/usr/share/wazuh-dashboard/src/core
/server/http/router/response_adapter.js:79:17)\n at Router.handle (/usr/share/wazuh-dashboard/src/core/server/http/router/router.js:175:34)\n at processTicksAndRejections (node:in
ternal/process/task_queues:95:5)\n at handler (/usr/share/wazuh-dashboard/src/core/server/http/router/router.js:140:50)\n at exports.Manager.execute (/usr/share/wazuh-dashboard/no
de_modules/@hapi/hapi/lib/toolkit.js:60:28)\n at Object.internals.handler (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/handler.js:46:20)\n at exports.execute (/usr/shar
e/wazuh-dashboard/node_modules/@hapi/hapi/lib/handler.js:31:20)\n at Request._lifecycle (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/request.js:371:32)\n at Request._ex
ecute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/request.js:281:9)"},"url":"https://MY_IP_ADDRESS/utils/logs/ui","message":"Internal Server Error"}
Oct 11 16:50:18 srv-wazuh opensearch-dashboards[702]: {"type":"response","@timestamp":"2024-10-11T15:50:18Z","tags":[],"pid":702,"method":"post","statusCode":500,"req":{"url":"/utils/lo
gs/ui","method":"post","headers":{"host":"MY_IP_ADDRESS","connection":"keep-alive","content-length":"1086","sec-ch-ua-platform":""Windows"","sec-ch-ua":""Google Chrome";v="129",
"Not=A?Brand";v="8", "Chromium";v="129"","sec-ch-ua-mobile":"?0","id":"default","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chr
ome/129.0.0.0 Safari/537.36","accept":"application/json, text/plain, /","content-type":"application/json","pattern":"wazuh-alerts-*","osd-xsrf":"kibana","origin":"https://MY_IP_ADDRES
S","sec-fetch-site":"same-origin","sec-fetch-mode":"cors","sec-fetch-dest":"empty","referer":"https://MY_IP_ADDRESS/app/endpoints-summary","accept-encoding":"gzip, deflate, br, zstd","a
ccept-language":"pt-PT,pt;q=0.9,en-US;q=0.8,en;q=0.7"},"remoteAddress":"REMOTE_ACCESS_IP_ADDRESS","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrom
e/129.0.0.0 Safari/537.36","referer":"https://MY_IP_ADDRESS/app/endpoints-summary"},"res":{"statusCode":500,"responseTime":14,"contentLength":9},"message":"POST /utils/logs/ui 500 14ms
9.0B"}

NOTE: I changed my ip's to MY_IP_ADDRESS for the server and REMOTE_IP_ADDRESS to the computer accessing the dashboards page.

**Comment by DMJoh** [comment]
Have you managed to get around this ?
![Uploading Screenshot 2024-10-31_Wazuh.png…]()


I am also having the same issue as you mentioned on 4.9.1. I have tested this with docker compose and deployed an OVA with version 4.9.1. Both the instances have this issue. Then I tried this on 4.8.2 and it was working there.

From logs,

```
{
  "type": "error",
  "@timestamp": "2024-10-31T04:49:15Z",
  "tags": [],
  "pid": 54,
  "level": "error",
  "error": {
    "message": "Internal Server Error",
    "name": "Error",
    "stack": "Error: Internal Server Error\n    at HapiResponseAdapter.toError (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:127:19)\n    at HapiResponseAdapter.toHapiResponse (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:83:19)\n    at HapiResponseAdapter.handle (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:79:17)\n    at Router.handle (/usr/share/wazuh-dashboard/src/core/server/http/router/router.js:175:34)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at handler (/usr/share/wazuh-dashboard/src/core/server/http/router/router.js:140:50)\n    at exports.Manager.execute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/toolkit.js:60:28)\n    at Object.internals.handler (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/handler.js:46:20)\n    at exports.execute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/handler.js:31:20)\n    at Request._lifecycle (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/request.js:371:32)\n    at Request._execute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/request.js:281:9)"
  },
  "url": "https://wz.domain.com/utils/logs/ui",
  "message": "Internal Server Error"
}
```

**Comment by WavedirectTel** [comment]
Same issue,  brand new install using ubuntu 24 using the installation scripts.

**Comment by himuura** [comment]
> Have you managed to get around this ?
> 
> ![Screenshot 2024-10-31_Wazuh](https://private-user-images.githubusercontent.com/6606893/382240649-be70f956-3b17-472a-9ab2-abba080b32c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NzA4MjksIm5iZiI6MTczMDQ3MDUyOSwicGF0aCI6Ii82NjA2ODkzLzM4MjI0MDY0OS1iZTcwZjk1Ni0zYjE3LTQ3MmEtOWFiMi1hYmJhMDgwYjMyYzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDFUMTQxNTI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Yzc0NTg4N2FjMDFiZGRhNzI4ODNlNzc5MWI4MGUyYWMzNTg3MzdhOTEyMjRiNjRjMGQ2YTBhZTAyMTM4OGI2NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.mebR_ZsOpHQIahJOnq5iP85PuW9vCefT6KWFHxHlgwI)
> 
> I am also having the same issue as you mentioned on 4.9.1. I have tested this with docker compose and deployed an OVA with version 4.9.1. Both the instances have this issue. Then I tried this on 4.8.2 and it was working there.
> 
> From logs,
> 
> ```
> {
>   "type": "error",
>   "@timestamp": "2024-10-31T04:49:15Z",
>   "tags": [],
>   "pid": 54,
>   "level": "error",
>   "error": {
>     "message": "Internal Server Error",
>     "name": "Error",
>     "stack": "Error: Internal Server Error\n    at HapiResponseAdapter.toError (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:127:19)\n    at HapiResponseAdapter.toHapiResponse (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:83:19)\n    at HapiResponseAdapter.handle (/usr/share/wazuh-dashboard/src/core/server/http/router/response_adapter.js:79:17)\n    at Router.handle (/usr/share/wazuh-dashboard/src/core/server/http/router/router.js:175:34)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at handler (/usr/share/wazuh-dashboard/src/core/server/http/router/router.js:140:50)\n    at exports.Manager.execute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/toolkit.js:60:28)\n    at Object.internals.handler (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/handler.js:46:20)\n    at exports.execute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/handler.js:31:20)\n    at Request._lifecycle (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/request.js:371:32)\n    at Request._execute (/usr/share/wazuh-dashboard/node_modules/@hapi/hapi/lib/request.js:281:9)"
>   },
>   "url": "https://wz.domain.com/utils/logs/ui",
>   "message": "Internal Server Error"
> }
> ```

Not yet my dude, but at least I'm not the only one. I'm hoping someone from the dev wazuh team to pick this up. It has something to do with the indexer and dashboard IMO, the query being made by the wazuh dashboard must be wrongly constructed or the indexer doesn't like the way it's being made. But no solution Soo far...

**Comment by himuura** [comment]
> Same issue, brand new install using ubuntu 24 using the installation scripts.

Thought it might have been me, I updated without reading the instructions but if a brand new install throws the same error, might be a bug somewhere in dashboard/indexer query.

**Comment by DawnMathewsJohn** [comment]
It seems this issue is also present in a fresh install of the new release (4.9.2). Isn’t it only affecting certain users? This is clearly a significant bug that’s unlikely to go unnoticed.

**Comment by asteriscos** [comment]
Thanks for reporting this. The mentioned component has a bug that has been fixed for Wazuh 4.10.0:
- https://github.com/wazuh/wazuh-dashboard-plugins/issues/7064

This component is designed to offer shortcuts to the Mitre ATT&CK dashboard filtering by the selected tactic. Until the fix is released, this can be achieved manually navigating to Mitre ATT&CK and applying the desired filter.
